src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -1058,10 +1400,11 @@ public enum RisingRivals implements LogicCardInfo {
           weakness P
           move "Recover", {
             text "Energy attached to Alakazam and remove 4 damage counters from Alakazam ."
-            energyCost P, P
+            energyCost P
             attackRequirement {}
             onAttack {
-              damage 0
+              discardSelfEnergy P
+              heal 40, self
```

Comment:
How is this supposed to work if I metronome? Should it always heal even if I can't discard a P energy?

Reply:
Looks like this is fine based on unofficial ruling.